base/interactiveutil.jl

```diff
@@ -571,11 +571,10 @@ end
 downloadcmd = nothing
 if is_windows()
     function download(url::AbstractString, filename::AbstractString)
-        res = ccall((:URLDownloadToFileW,:urlmon),stdcall,Cuint,
-                    (Ptr{Void},Cwstring,Cwstring,Cuint,Ptr{Void}),C_NULL,url,filename,0,C_NULL)
-        if res != 0
-            error("automatic download failed (error: $res): $url")
-        end
+        ps = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
```

Comment:
I believe powershell will always be in path so it may be best to remove the explicit location?